resolv.cabal

```diff
@@ -68,7 +68,6 @@ tested-with:
   -- GHC == 7.10.3
   -- GHC == 7.8.4
   -- GHC == 7.6.3
```

Comment:
LGTM, tho, to follow up, the stats show <2% of users are GHCs <8.4 as of the last Yearly Haskell Survey in 2021, which is a further diminishing category as time goes on (esp. with respect to the 7.x series). Considering that the lower bound cap for `base` places you outside of the large number of performance improvements seen in `bytestring-0.11`, I'm not sure that keeping these old GHC versions is the prudent choice. The rest of the ecosystem, as well as most of the core libraries, now require a GHC of at least 8.x. What do you think about dropping the older GHCs to bring the library in line with what you can reasonably test?

Reply:
I hear your message.
I am happy to drop any GHC 7.x if it brings trouble.  E.g., should `resolv` get new functionality which needs testing or wants to use newer GHC features, there would be no reason to hang on to ancient GHCs.  But I'd drop these if there is need; otherwise, while `resolv` is stable, I see no harm in keeping them around.